### PR TITLE
translated into Telugu locale

### PR DESCRIPTION
**Pull request by spathak98**

### CODE REVIEW

File: app/config/locale/translations/tel.countries.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    'LR' => 'లైబీరియా',
+    'LY' => 'లిబియా',
+    'LC' => 'సెయింట్ లూసియా',
+    'LI' => 'లిచ్టెన్స్టెయిన్',
+    'LK' => 'శ్రీలంక',
+    'LS' => 'లెసోతో',
+    'LT' => 'లిథువేనియా',
+    'LU' => 'లక్సెంబర్గ్',
+    'LV' => 'లాట్వియా',
+    'MA' => 'మొరాకో, వెస్ట్, సూర్యాస్తమయం',
```
Comment (spielers) [inline app/config/locale/translations/tel.countries.php:110]: the 'మొరాకో, వెస్ట్, సూర్యాస్తమయం', you should remove the comma from this

File: app/config/locale/translations/tel.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    'settings.locale' => 'tel',
+    'settings.direction' => 'rtl',
+
+    // Service - Users
+    'account.emails.team' => 'జట్టు %s',
+    'account.emails.verification.title' => 'ఖాతా నిర్ధారణ',
+    'account.emails.verification.body' => 'tel.email.auth.confirm.tpl',
+    'account.emails.recovery.title' => 'పాస్వర్డ్ రీసెట్',
+    'account.emails.recovery.body' => 'tel.email.auth.recovery.tpl',
+    'account.emails.invitation.title' => 'జట్టు కోసం కాల్ చేయండి %s a లో %s',
```
Comment (spielers) [inline app/config/locale/translations/tel.php:14]: recheck the translation once the way of %s donation has misplaced

**Review by spielers — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/tel.countries.php (hunk 1 above)
Comment (spathak98) [reply app/config/locale/translations/tel.countries.php]: @spielers I have done suggested changes, please look that changes.

File: app/config/locale/translations/tel.php (hunk 2 above)
Comment (spathak98) [reply app/config/locale/translations/tel.php]: @spielers I have done suggested changes, please look that changes.

**Comment by eldadfux** [comment]
@spielers are that last changes OK with you?

@spathak98 please resolve conflict before I can megre.

**Comment by spielers** [comment]
> @spielers are that last changes OK with you?
> 
> @spathak98 please resolve conflict before I can merge.

Suggested Few changes @spathak98  Resolve the conflict will get you merged

**Comment by spathak98** [comment]
@spielers I have resolved the issue, Please look in to it.

**Review by spielers — APPROVED** [review]
Hey, @spathak98 thanks for your contributions will get you merged!

Best,
spielers

**Comment by eldadfux** [comment]
@spathak98 One issue before merging. Please use a 2-letter code and not a 3-letter code for the new language 🙏🏻

**Comment by eldadfux** [comment]
You can use `te` code as listed in ISO-639-1: https://gist.github.com/joshuabaker/d2775b5ada7d1601bcd7b31cb4081981

**Pull request closed without merging**